src/java.base/share/classes/java/net/ServerSocket.java

```diff
@@ -518,7 +518,7 @@ public SocketAddress getLocalSocketAddress() {
      * client socket implementation factory}, if one has been set.
      *
      * @throws     IOException  if an I/O error occurs when waiting for a
-     *               connection.
+     *               connection, or if the socket isn't bound or is already closed.
```

Comment:
Can you check the existing text as I think it uses "not bound" rather than "isn't bound".

Reply:
Also "already close" might be a bit inconsistent with the existing API docs too, can you check as I think we use  "is closed" in most places.

Reply:
Hello Alan, I've now updated the PR to address this text.

Reply:
Thanks, I can review the CSR when it's ready.

Reply:
Updated docs changes look good.